src/lib/containers/Login.tsx

```diff
@@ -172,42 +165,42 @@ class Login extends React.Component<Props, State> {
           <img
             height="20px"
             style={{ marginRight: '10px' }}
-            alt={'sage bionetworks logo'}
+            alt={'Sage Bionetworks logo'}
             src="https://s3.amazonaws.com/static.synapse.org/sage-bionetworks-logo.svg"
           />
           Sign in with your Sage Bionetworks Synapse account
         </div>
-        <form onSubmit={this.handleLogin} className="form-group">
-          <input
+        <Form onSubmit={this.handleLogin}>
+          <FormControl
             autoComplete="username"
             placeholder="username or email"
-            className="form-control SRC-marginBottomTop"
+            className="LoginComponent__Input"
             id="exampleEmail"
             name="username"
             type="text"
             value={this.state.username}
             onChange={this.handleChange}
           />
-          <input
+          <FormControl
             autoComplete="password"
             placeholder="password"
-            className="form-control SRC-marginBottomTop"
+            className="LoginComponent__Input"
             id="examplePassword"
             name="password"
             type="password"
             value={this.state.password}
             onChange={this.handleChange}
           />
           {this.getLoginFailureView()}
-          <button
+          <Button
```

Comment:
Subtle, but the text in the Google button is also no longer slightly offset.

https://user-images.githubusercontent.com/17580037/104050602-db4aaf00-51b4-11eb-8abf-0c1baef2d59e.mov